config/rector/sets/cakephp37.php

```diff
@@ -45,10 +45,10 @@
         new PropertyFetchToMethodCall('Cake\View\View', 'helpers', 'helpers'),
     ]);
 
-    $rectorConfig->ruleWithConfiguration(MethodCallToAnotherMethodCallWithArgumentsRector::class, [
-        new MethodCallToAnotherMethodCallWithArguments('Cake\Database\Query', 'join', 'clause', ['join']),
-        new MethodCallToAnotherMethodCallWithArguments('Cake\Database\Query', 'from', 'clause', ['from']),
-    ]);
+    //$rectorConfig->ruleWithConfiguration(MethodCallToAnotherMethodCallWithArgumentsRector::class, [
+    //    new MethodCallToAnotherMethodCallWithArguments('Cake\Database\Query', 'join', 'clause', ['join']),
+    //    new MethodCallToAnotherMethodCallWithArguments('Cake\Database\Query', 'from', 'clause', ['from']),
+    //]);
```

Comment:
Its easier to just disable these rector rules.
We can't represent those with the new `RenameMethodRector` and `AddMethodCallArgsRector` since 
```
Cake\Database\Query::join();
Cake\Database\Query::from();
```
would need to be transformed into
```
Cake\Database\Query::clause('join');
Cake\Database\Query::clause('from');
```
and rector doesn't allow to define multiple configurations inside one ruleset.

Reply:
We should add a comment explaining why they're disabled.